it/resources/cdcm/create-conversation-no-email.json

```diff
@@ -12,6 +12,10 @@
         "value": "GB1234567890"
```

Comment:
-no-email.json makes no sense, as there is no way to provide an email, is this the same case as -full.json ?